src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -1981,7 +1981,7 @@ TEST(SpatialPoolerTest, testSerialization2) {
     {
       SpatialPooler spTemp;
 
-      testTimer.start();
+      testTimer -= clock();
```

Comment:
use `getElapsed()` from the new utils/Time.hpp

Reply:
Ok, I can rewrite it as follows:
```
double testTimer = 0 // seconds
for(){
  clock_t start_time = clock()
  -do-work-
  testTimer += getElapsed(start_time)
}
```